modules/claims_api/app/services/claims_api/dependent_claimant_poa_assignment_service.rb

```diff
@@ -131,7 +131,7 @@ def dependent_claims
 
       log(level: :error, detail: 'Dependent claims not found in BGS')
 
-      raise ::Common::Exceptions::ResourceNotFound
+      {}
```

Comment:
removed raise so later code executes and returned {} so it plays nice with .find later on. I can change this to return nil then check for it if we'd prefer that